src/index.tsx

```diff
@@ -157,12 +159,13 @@ function LazyHydrate(props: Props) {
     whenVisible,
     didHydrate,
     promise,
-    noWrapper
+    noWrapper,
+    as
```

Comment:
This effect does not depend on `as`.
```suggestion
    noWrapper
```